dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoryDTO.java

```diff
@@ -0,0 +1,123 @@
+package com.dotcms.rest.api.v1.categories;
+
+import java.util.Date;
+
+public class CategoryDTO {
+
+    private String categoryName;
```

Comment:
we do not allow non-immutables beans, is there any special reason to create this as a mutable

Reply:
as such no reason - let me make it immutable :)